.gitignore

```
@@ -35,3 +35,5 @@ out/
 prod.env
 minio_data/
 !javadoc/
+/build/generated/sources/
+/build/generated/
```

```suggestion
build/
!build/generated-snippes/
```
would be more elegant